package.json

```diff
@@ -1,12 +1,9 @@
 {
-  "name": "broccoli-builder",
+  "name": "ember-cli-broccoli",
```

Comment:
is this what we want or do we want to change this to `broccoli`?

Reply:
Ah this was unintentional.